src/monitor/pgautofailover--1.5--1.6.sql

```diff
@@ -4,6 +4,96 @@
 -- complain if script is sourced in psql, rather than via CREATE EXTENSION
 \echo Use "CREATE EXTENSION pgautofailover" to load this file. \quit
 
+ALTER TABLE pgautofailover.node
+	RENAME TO node_upgrade_old;
+
+ALTER TABLE pgautofailover.node_upgrade_old
+      RENAME CONSTRAINT system_identifier_is_null_at_init_only
+                     TO system_identifier_is_null_at_init_only_old;
+
+ALTER TABLE pgautofailover.node_upgrade_old
+      RENAME CONSTRAINT same_system_identifier_within_group
+                     TO same_system_identifier_within_group_old;
+
+CREATE TABLE pgautofailover.node
+ (
+    formationid          text not null default 'default',
+    nodeid               bigserial,
```

Comment:
Should be using the `pgautofailover.node_nodeid_seq` sequence, now it creates its own one called `pgautofailover.node_nodeid_seq1`.

Reply:
Fixed now, good catch here! I applied your `\dx+` trick and got the expected diff now between the v1.5.2 and the 1.6.0 in development:

``` diff
diff /tmp/objs.list.v1.5.2.txt /tmp/objs.list.pr.txt
12,13c12,13
<  function pgautofailover.get_other_nodes(integer)
<  function pgautofailover.get_other_nodes(integer,pgautofailover.replication_state)
---
>  function pgautofailover.get_other_nodes(bigint)
>  function pgautofailover.get_other_nodes(bigint,pgautofailover.replication_state)
18c18
<  function pgautofailover.node_active(text,integer,integer,pgautofailover.replication_state,boolean,pg_lsn,text)
---
>  function pgautofailover.node_active(text,bigint,integer,pgautofailover.replication_state,boolean,integer,pg_lsn,text)
21,22c21,22
<  function pgautofailover.register_node(text,text,integer,name,text,bigint,integer,integer,pgautofailover.replication_state,text,integer,boolean,text)
<  function pgautofailover.remove_node(integer)
---
>  function pgautofailover.register_node(text,text,integer,name,text,bigint,bigint,integer,pgautofailover.replication_state,text,integer,boolean,text)
>  function pgautofailover.remove_node(bigint)
29,30c29,30
<  function pgautofailover.start_maintenance(integer)
<  function pgautofailover.stop_maintenance(integer)
---
>  function pgautofailover.start_maintenance(bigint)
>  function pgautofailover.stop_maintenance(bigint)
```